src/style.css

```diff
@@ -2,0 +1,66 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+  font-weight: 900;
+  font-style: normal;
+}
+
+.search__form {
+  margin-top: 20px;
```

Comment:
Be consistent with your margins. Add only top or bottom, but not both. You've used `margin-top` here, stick to it throughout your CSS.